reference/api/tasks.md

```diff
@@ -117,17 +117,23 @@ This value is always `null` for `dumpCreation` tasks.
 
 #### `taskDeletion`
 
+| Name                  | Description                                                                                                                                                                             |
+| :-------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`matchedTasks`**    | The number of matched tasks based on the request. If the API key doesn’t have access to any of the indexes specified in the request, those tasks will not be included in `matchedTasks` |
```

Comment:
I've made a comment on a similar paragraph on #1981 that also applies here: https://github.com/meilisearch/documentation/pull/1981#discussion_r1021994151